CMakeLists.txt

```diff
@@ -3,7 +3,14 @@
 
 cmake_minimum_required(VERSION 3.8.0)
 
-# Fortran below is needed for SPHEREPACK.
+# Set up the project. Notes:
+# - Scripts may automatically regex-replace the version string in this command.
+#   For example, here's how you could set the version to 'X.Y.Z':
+#
+#   sed -i 's/^\(project(.*VERSION \)\([^ )]\)*/\1X.Y.Z/' CMakeLists.txt
```

Comment:
It looks like this PR moves us away from X.Y.Z versioning, so that's probably a bad example.